pkg/storkctl/clusterpair.go

```diff
@@ -249,14 +252,22 @@ func newCreateClusterPairCommand(cmdFactory Factory, ioStreams genericclioptions
 				util.CheckErr(err)
 				return
 			}
-
-			srcClusterPair, err := generateClusterPair(clusterPairName, cmdFactory.GetNamespace(), dIP, dPort, destToken, dFile, projectMappingsStr, false)
+			fmt.Printf("Using PX-Service Endpoint of DR cluster to create clusterpair...\n")
```

Comment:
If this is for user output, why not use printMsg like elsewhere in this function?